### PR TITLE
Clean ups - No mixed annotations

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: tests/Fixer/Import/OrderedImportsFixerTest.php [hunk 1]
```diff
@@ -1594,14 +1594,15 @@ public function provide70TypesOrderAndLength()
...
      * @requires PHP 7.0
      *
-     * @param mixed $expected
-     * @param mixed $input
+     * @param string      $expected
+     * @param string|null $input
+     * @param string[]    $importOrder
      */
-    public function test70TypesOrderAndAlphabet($expected, $input = null)
+    public function test70TypesOrderAndAlphabet($expected, $input = null, array $importOrder = null)
```
Comment (SpacePossum) [inline tests/Fixer/Import/OrderedImportsFixerTest.php]: prep. for missing tests for case described here https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2696

**Comment by keradus** [comment]
cool changes.
is it WIP ?

**Comment by SpacePossum** [comment]
IMO; this is good to go :)

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**